packages/jest-snapshot/src/utils.js

```diff
@@ -76,14 +76,18 @@ const ensureDirectoryExists = (filePath: Path) => {
   } catch (e) {}
 };
 
+// Normalize newlines on different OS Platforms
```

Comment:
this comment is superfluous; the function name is already "normalizeNewline". Please remove it :)